File: Module06/bank-account-comparison/BankDemo.java
package edu.au;

public class BankDemo {
    public static void  main(String[] argv) {
        BankAccount account = new BankAccount(123456, "Fred");
		System.out.println(account);
        System.out.println(account.getOwner());
        account.setOwner("Betty");
		System.out.println(account);
        //account.setOwner("f");
        account.setOwner(null);
    }
}

File: Module06/bank-account-comparison/bank_demo.py
from bank_account import BankAccount


def main() -> None:
    account = BankAccount(123456, "Fred")
    print(account)
    print(account.owner)
    account.owner = "Betty"
    print(account)
    # account.owner = "f"
    # account.owner = None
    print(account)
    print(account.balance)
    account.account_number = 99


if __name__ == "__main__":
    main()
